sdk/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
   kotlin("android")
   id("com.jaredsburrows.license")
   id("org.jetbrains.dokka")
+  id("com.mapbox.android.sdk.versions")
```

Comment:
currently we only apply the version plugin to the sdk module, however we could discuss if we want to add it to all our plugins, as mentioned in https://github.com/mapbox/android-sdk-versions-plugin#about-android-sdk-versions-plugin
> All Mapbox android SDK modules that incorporate the Telemetry SDK and meant to be released as libraries(artifacts) should apply this plugin to facilitate the Telemetry SDK access all Mapbox library versions at run time.

Reply:
As discussed offline with @nagineni 
> we should be adding the dependency only to the core module.

We can discuss more if we want to add it to plugins as well, but we will iterate this in future releases. currently, core doesn't have a parsing logic in place and they are working on another solution as well.